Cargo.toml

```diff
@@ -66,7 +66,7 @@ members = [
 [workspace.dependencies]
 imxrt-boot-gen = { version = "0.3.0", features = ["imxrt1060"] }
 imxrt-hal = { version = "0.5.3", features = ["imxrt1060"] }
-imxrt-iomuxc = { version = "0.2.0", features = ["imxrt1060"] }
+imxrt-iomuxc = { version = "0.2.9", features = ["imxrt1060"] }
```

Comment:
Even with this LPUART5 and 7 addition, `teensy4-bsp` can build its examples with a minimum version of imxrt-iomuxc 0.2.1. Sorry, but I hesitate to force users to adopt a newer version of this package when it doesn't seem necessary.

I removed this from the commit. If you could clarify why we should bump this, I'm happy to reconsider!